model/annotation_test.go

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+//
+
+package model
+
+import (
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+func TestAnnotationsFromStringSlice(t *testing.T) {
+
+	t.Run("valid annotations", func(t *testing.T) {
```

Comment:
We need some valid test cases that include the one allowed period by the regex

Reply:
As mentioned in the comment above it was not intended to allow period in annotations.